src/Flame/Html/Element.js

```diff
@@ -167,9 +167,12 @@ exports.createSingleSvgNode = function (nodeData) {
 };
 
 function asSvg(elements) {
-    for (let e of elements)
+    for (let e of elements) {
         if (e.nodeType === elementNode)
             e.nodeType = svgNode;
+        if (typeof e.children !== 'undefined')
```

Comment:
This won't protect against ```null```, will it? I think on JavaScript files we gotta check for both.

Reply:
I got no errors on any of my SVG making use of all of `element`, `element'`, and `element_`. If you can tell me how make `children` null I can test it. `typeof null` in the console does give `'object'` so yeah it should fail if children can actually be null and not just undefined.

Reply:
Users of the library can't make ```children``` to be ```null```. Someone modifying the code in the future could :)